Automattic/jetpack
# Make wordads discoverable

Fixes https://github.com/Automattic/jetpack/issues/33165

## Proposed changes:

This change will alow registering Wordads Block even if the Wordads module is disabled. It should show the placeholder allowing to enable the module in one click. This is aimed at making our blocks more discoverable.

Note: it will only show up if user owns a plan that allow WordAds.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Spin up the branch locally and tunnel with JT to your sandbox
* Build `plugins/jetpack`
* Navigate to `/wp-admin/admin.php?page=jetpack_modules` and make sure that Ads module is disabled
* Make sure you don't have plan that supports Ads (No Complete)
* Create a new post `/wp-admin/post-new.php`
* In blocks menu make sure you **don't** see the `Ad` block.
* Purchase 'Jetpack Complete' plan
* Create a new post `/wp-admin/post-new.php` and you should see 'Ad' block.
* Adding it should show loading and/or a placeholder with the ability to activate the module. 

<img width="680" alt="Screenshot at Sep 14 15-35-30" src="https://github.com/Automattic/jetpack/assets/60262784/c7ed4c14-19b6-43d3-8d5b-f35960d4232c">

<img width="658" alt="Screenshot at Oct 19 18-48-39" src="https://github.com/Automattic/jetpack/assets/60262784/d4e0cbfe-a07b-4b1d-afa2-e1bdd9a910ca">

* Activate the module and check that the form works as expected.
